commands/report.js

```
@@ -54,46 +54,95 @@ async function report (argv, _dir) {
+  } else {
+    // When using local ncm-ng, attempt to load local whitelist config if available
+    try {
+      const fs = require('fs')
+      const path = require('path')
+      const localWhitelistPath = path.join(process.cwd(), '.ncm-whitelist.json')
+      
+      if (fs.existsSync(localWhitelistPath)) {
+        const localWhitelist = JSON.parse(fs.readFileSync(localWhitelistPath, 'utf8'))
+        if (Array.isArray(localWhitelist)) {
+          for (const item of localWhitelist) {
+            if (item && item.name && item.version) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)